mifospay/src/main/res/layout/activity_settings.xml

```diff
@@ -43,19 +43,30 @@
         android:id="@+id/btn_disable_account"
         android:layout_width="match_parent"
         android:layout_height="60dp"
-        android:layout_marginTop="@dimen/value_20dp"
+        android:layout_marginStart="8dp"
```

Comment:
I think it would be better if the margin is a dimensions file resource rather than a hard-coded value

Reply:
Yes, I will do it.

Reply:
Done. Please review.